README.md

```diff
@@ -53,7 +53,47 @@ cd onyo
 pip install -e ".[tests, docs]"
 ```
 
-### Tests
+### Design
+
+Onyo follows a set of development and coding conventions throughout its code
+base. For linting and type checking, the Onyo project uses flake8 and Pyre. The
+following sections describe additional conventions followed in this project,
+extending the existing [PEP 8 Style Guide](https://peps.python.org/pep-0008/).
```

Comment:
"Extending" sounds like we've written an extension to PEP 8. I think that's a bold claim, beyond the scope of what's written here. ;-)